vdo: document versions that have read cache options

bgurney-rh opened the pull request:
The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.  Document that the read
cache options are only in VDO 6.1 and older.

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
The read cache feature in VDO has been removed in VDO version 6.2.  No code needs to be changed; instead, add documentation lines to the "readcache" and "readcachesize" options, stating that they are available in VDO versions 6.1 and older, allowing those versions to continue to use playbooks with those options.

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->
vdo

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->
If playbooks for systems with VDO 6.2 and newer do not specify the "readcache" or "readcachesize" options, there should not be any errors.  If either of these options are specified for VDO 6.2 and above, there will be an error from the "vdo" command stating "unrecognized arguments: --readCache..." and/or "unrecognized arguments: --readCache..."; these errors should be passed through by ansible, and visible to the user.
<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

hvtuananh committed:
Used C locale instead of relying on system locale.

bgurney-rh committed:
vdo: document versions that have read cache options

The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

bgurney-rh committed:
vdo: document versions that have read cache options

The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

bgurney-rh committed:
vdo: document versions that have read cache options

The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

bgurney-rh committed:
vdo: document versions that have read cache options

The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

NilashishC committed:
Revert back to using section (#53069)

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

willthames committed:
aws_kms tags fix (#53088)

* Fix TagName and TagValue in aws_kms

Fixes #53061

* Improve test suite to check for tags

Also fixed some obvious failures, need to run the test suite from time
to time!

s-hertel committed:
Enable ec2_vpc_vgw tests (#53118)

WojciechowskiPiotr committed:
Documentation update for labels operations on swarm/node (#53083)

* Update lib/ansible/modules/cloud/docker/docker_node.py
* Update lib/ansible/modules/cloud/docker/docker_swarm.py

Co-Authored-By: WojciechowskiPiotr <23406016+WojciechowskiPiotr@users.noreply.github.com>

jtyr committed:
Last two fields in fstab are optional (fixes #43855) (#43941)

* Last two fields in fstab are optional
* Add changelog

tedder committed:
botmeta: ignore tedder, fix some grammar (#53133)

* ignore tedder, fix some grammar

I'm less helpful than in the past on reviewing these, adding myself to the 'ignore' keys.

Also fixed some grammar in the instruction block.

* oh, so that's how the default key works. yaml fail

dagwieers committed:
uri/win_uri: Make method a free text field (#49719)

* uri/win_uri: Make method a free text field

Since various interfaces introduce their own HTTP method (e.g. like
PROPFIND, LIST or TRACE) it's better to leave this up to the user.

* Fix HTTP method check in module_utils urls

* Add integration test for method UNKNOWN

* Clarify the change as requested during review

samdoran committed:
Reboot - add parameter for paths to search for shutdown command (#51194)

* Look in /lib/molly-guard for shutdown command

Signed-off-by: Sam Doran <sdoran@redhat.com>

* Add parameter for extra_search_paths

Signed-off-by: Sam Doran <sdoran@redhat.com>

* Change option to search_paths

- Update docs
- Make the parameter replace the default values
- Add some sanity checking on the data type

Signed-off-by: Sam Doran <sdoran@redhat.com>

* Correct parameter in tests, change conditional for molly-guard tasks

Signed-off-by: Sam Doran <sdoran@redhat.com>

* Address feedback

* Simplify field validation

Needed the try except in case a non-iterable type is put in the search_paths field

samdoran committed:
Move _handle_aliases() out of basic.py (#48578)

Refinements:
- return legal_inputs and update class properties
- remove redundant arguments from method and handle in caller
- add better exception types to method

* Add unit tests for handle_aliases

s-hertel committed:
replace uses of sort_json_policy_dict with compare_policies (#52943)

* replace uses of sort_json_policy_dict with compare_policies which is compatible with Python 3 and remove sort_json_policy_dict from the AWS guidelines since it only works on Python 2

* Sort any lists containing only strings

* Sort the original policy too

sivel committed:
Don't allow an empty version_added

nitzmahone committed:
interpreter discovery message tweaks (#53155)

* fixes #53120 (doclink generator now understands devel)
* added hosts to some warnings

mattclay committed:
Overhaul ansible-test cloud test plugins. (#53044)

acdha committed:
docs: SAN prefixes are required for openssl_csr (#53144)

The openssl_csr docs use “should” where (at least following IETF nomenclature) “must” would be more accurate.

See #53141 

+label: docsite_pr

yungezz committed:
fix keyvault auth in msi auth (#53100)

yungezz committed:
fix class name (#53162)

RedWhiteMiko committed:
Do not delete duplicate host when fetching Zabbix Hosts (#53160)

fanf2 committed:
get_url: return no change in check mode when checksum matches (#53070)

Signed-off-by: Tony Finch <dot@dotat.at>

machacekondra committed:
kubevirt: Fix cloud_init in kubevirt_vm (#52999)

mmazur committed:
Add kubevirt_cdi_upload module (#52990)

xmadsen committed:
Resolved conflicts with mistaken commits in earlier history (#52953)

xmadsen committed:
Remove mistaken commits from other pull request. (#52928)

migumun committed:
Fortinet's FortiOS wireless controller wtp (#52850)

* Fortinet's FortiOS wireless controller wtp

* Avoid using global

migumun committed:
Fortinet's FortiOS wireless controller wids profile (#52849)

* Fortinet's FortiOS wireless controller wids profile

* Avoid using global

migumun committed:
Fortinet's FortiOS wireless controller vap (#52847)

* Fortinet's FortiOS wireless controller vap

* Avoid using global

migumun committed:
Fortinet's FortiOS wireless controller utm profile (#52846)

* Fortinet's FortiOS wireless controller utm profile

* Avoid using global

migumun committed:
Fortinet's FortiOS wireless controller setting (#52845)

* Fortinet's FortiOS wireless controller setting

* Avoid using global

felixfontein committed:
Don't die when kernel memory accounting is not supported by runc build. (#53163)

rubentsirunyan committed:
Fixing mediatype default value (#53114)

* Fixing mediatype default value

* Updating the documentation of media_type

rubentsirunyan committed:
[zabbix_action] Refactoring the argument spec (#53106)

* Refactoring the argument spec

* Adding subarguments for recovery and acknowledge operations

* Correcting the linting errors

mkrizek committed:
profile_tasks: Fix last task time with multiple plays (#53065)

gundalow committed:
hpe3par document type (#52992)

* Update lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py

Co-Authored-By: gundalow <john@johnrbarker.com>

rubentsirunyan committed:
Zabbix action: Adding operation condition (#53173)

mattclay committed:
Remove invalid validate-modules ignore entry.

sivel committed:
validate-modules improve empty version_added handling (#53177)

* Improve empty version_added handling

* Remove unrelated ignore

s-hertel committed:
Disable ec2_vpc_vgw tests yet again (#53184)

arnaudmorin committed:
Fix typo (#53123)

I changed the typo: acceess to access

Commit:
openssl_certificate, fixed has_expired to check the cert expiration date (#53168)

mnecas committed:
add mnecas to BOTMETA.yml (#53130)

rubentsirunyan committed:
Zabbix action: Correcting 'notify_all_involved' for acknowledge operations (#53197)

wbagdon committed:
Add documentation for win_copy as become user needing temp dir set (#53211)

dagwieers committed:
mso_schema_template: Fix examples (#53237)

The examples were using the wrong module.

dagwieers committed:
mso_s_t_anp_epg: Fix incompatibility with newer MSO (#53238)

Due to a change with the backend contractRelationships is required.

dagwieers committed:
mso_st_filter_entry: Fix various issues (#53241)

This PR includes:
- Use the correct module name in the examples
- Small improvement in logic
- Fix an issue where updating an existing entry could reset to defaults

dagwieers committed:
MSO: Clean up internal references (#53242)

During the development of the new site modules the use of references is
more common and this urged making the mechanism more simple overall.

dagwieers committed:
mso_stae_subnet: Fix various issues (#53239)

This PR includes:
- Rename 'ip' to 'subnet' as that's what we're managing
- Use the correct module name in the examples
- Small improvement in logic

dagwieers committed:
mso_st_bd_subnet: Fix various issues (#53240)

This PR includes:
- Rename 'ip' to 'subnet' as that's what we're managing
- Small improvement in logic

dagwieers committed:
MSO: New module to manage local site ANPs (#53243)

This adds a new module to manage local site ANPs on ACI MultiSite schema
templates.

dagwieers committed:
MSO: New module to manage local site EPGs (#53244)

This adds a new module to manage local site EPGs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site static leafs (#53245)

This adds a new module to manage local site static leafs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site static ports (#53246)

This adds a new module to manage local site EPG static ports on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site EPG subnets (#53247)

This adds a new module to manage local site EPG subnets on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site BDs (#53248)

This adds a new module to manage local site BDs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site BD l3outs (#53249)

This adds a new module to manage local site BD l3outs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site EPG subnets (#53250)

This adds a new module to manage local site EPG subnets on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site VRF regions (#53252)

This adds a new module to manage local site VRF regions on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site VRFs (#53251)

This adds a new module to manage local site VRFs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site VRF region CIDRs (#53253)

This adds a new module to manage local site VRF region CIDRs on ACI MultiSite schema templates.

dagwieers committed:
MSO: New module to manage local site VRF region CIDR subnets (#53254)

This adds a new module to manage local site VRF region CIDR subnets on ACI MultiSite schema templates.

Commit:
udpating dtl version (#53220)

jborean93 committed:
Use setup_remote_tmp for windows httptester (#53255)

Commit:
Adding DevTest Lab Artifact Source facts module (#53194)

Commit:
Adding DevTest Lab Artifact facts (#53195)

ftntcorecse committed:
Fortinet FortiManager Module_Utils MINOR Update (#52756)

* PR for minor update to FortiManager.py.

Now accepts entire module as input, so it can use those data points for decisions later. This update is entirely related to our debugging and should have zero impact on existing modules.

* Minor change to documentation to kick off another shippable. Suspected error on last run.

* Commented out reference to the import library for fortimanager debug (which is only used by us, internally). It needs to be moved outside of the Ansible namespace so we can call it with try/catch without any other issues. Right now /ansible/executor/module_common.py runs w/ recursive_finder, and throws errors because it can't find debug on other machines within the ansible namespace. We don't have this issue with pyFMG legacy usage.

So, we're disabling the code until we have a better way of calling this debug module. It isn't required for production usage.

ftntcorecse committed:
FortiManager Plugin Module Conversion: fmgr_fwobj_address (#52773)

* Auto Commit for: fmgr_fwobj_address

* Auto Commit for: fmgr_fwobj_address

* Auto Commit for: fmgr_fwobj_address

* Auto Commit for: fmgr_fwobj_address

* Commit contains updates to module, unit test, and/or fixture files: fmgr_fwobj_address

ftntcorecse committed:
FortiManager Plugin Module Conversion: fmgr_query (#52770)

* Auto Commit for: fmgr_query

* Auto Commit for: fmgr_query

ftntcorecse committed:
FortiManager Plugin Module Conversion: fmgr_device (#52767)

* Auto Commit for: fmgr_device

* Auto Commit for: fmgr_device

ftntcorecse committed:
FortiManager Plugin Module Conversion: fmgr_device_config (#52765)

* Auto Commit for: fmgr_device_config

* autopep8 changes

* Auto Commit for: fmgr_device_config

* Auto Commit for: fmgr_device_config

migumun committed:
Fortinet's FortiOS wireless controller wtp profile (#52851)

* Fortinet's FortiOS wireless controller wtp profile

* Avoid using global

* Remove unnecessary code ('flatten' method)

NilashishC committed:
New module to support BGP configuration management in IOS (#49121)

* ios_bgp initial push

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Added tests for ios_bgp

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fixed docs

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Added space

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix Shippable Errors

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix Shippable Errors

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add support for af_neighbor option

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add support for af_neighbor option - 2

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add support for af_neighbor option - 3

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix typo

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Refactor BGP

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI and previous reviews

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add missing params documentation

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Remove previous tests

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Remove elements=dict from keys with type=list from args spec for element validation to pass

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Added function to validate input

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix sanity test failure

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Minor bug fixes

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix typo in fixture

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add integration tests

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

migumun committed:
Fortinet's FortiOS wireless controller global (#52844)

* Fortinet's FortiOS wireless controller global

* Avoid using global

* Updated from comments received

yuwzho committed:
Add addon/network/aad support for AKS (#51014)

Commit:
Adding DevTest Lab ARM Template facts (#53196)

Commit:
VMware: Update vmware_object_role_permission to allow permissions at root folder (#50465)

* Update vmware_object_role_permission to allow permissions at root folder
* Add example for adding to root folder

NilashishC committed:
frr: New module for BGP configuration management (#51281)

* Added frr_bgp module

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix pycodestyle warnings

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI docs issues

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Initial test files

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix dir structure to make CI happy

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix imports

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Split neighbor spec to handle general session and af session

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* New set of unittests

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix CI - 2

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Update license

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add port option for neighbor

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Validate network input

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Validate network input

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Fix sanity test failure

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

* Add validation for AF and more examples

Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

ftntcorecse committed:
New FortiManager Module: fmgr_fwobj_ippool6 (#53032)

* Auto Commit for: fmgr_fwobj_ippool6

* Auto Commit for: fmgr_fwobj_ippool6

ftntcorecse committed:
New FortiManager Module: fmgr_fwobj_ippool (#53031)

* Auto Commit for: fmgr_fwobj_ippool

* Auto Commit for: fmgr_fwobj_ippool

ftntcorecse committed:
New FortiManager Module:  fmgr_secprof_voip (#53030)

* Auto Commit for: fmgr_secprof_voip

* Auto Commit for: fmgr_secprof_voip

ftntcorecse committed:
New FortiManager Module:  fmgr_secprof_proxy (#53029)

* Auto Commit for: fmgr_secprof_proxy

* Auto Commit for: fmgr_secprof_proxy

* Auto Commit for: fmgr_secprof_proxy

ftntcorecse committed:
New FortiManager Module:  fmgr_secprof_appctrl (#53028)

* Auto Commit for: fmgr_secprof_appctrl

* Auto Commit for: fmgr_secprof_appctrl

felixfontein committed:
docker_swarm: support older docker-py versions (#53129)

* Decreasing docker_swarm requirements.

* Fixing docker-py / docker API version requirements, and some comments.

* Add changelog.

* Only send parameters specified by user to docker daemon.

* Extend labels test: not specifying == keep labels.

* Bump minimally required docker-py version for docker_node and docker_node_facts to 2.4.0.

* Prevent crashing when publish or healthcheck is not provided.

* Similarly to docker_swarm tests, only execute docker_node tests on real VMs and restart docker daemon when tests are done.

ftntcorecse committed:
New FortiManager Module: fmgr_fwobj_service (#53027)

* Auto Commit for: fmgr_fwobj_service

* Auto Commit for: fmgr_fwobj_service

ftntcorecse committed:
New FortiManager Module: fmgr_fwpol_package (#53026)

* Auto Commit for: fmgr_fwpol_package

* Auto Commit for: fmgr_fwpol_package

* Auto Commit for: fmgr_fwpol_package

Spredzy committed:
Become plugin: Update documentation to match other plugin doc (#53266)

7aitsev committed:
Fixed a typo in playbooks_async.rst (#53200)

sebwoj committed:
Remove redundant article in testing integration note (#53209)

Qalthos committed:
restconf_config module (#51971)

* Add restconf_config module

* Try to do the right thing when given partial paths

* Add PATCH

* Delete should not require content

* Non-JSON exceptions need raising, too

* Let ConnectionError objects pass through exec_jsonrpc

chrisgavin committed:
Fix an incorrect function call in a documentation example. (#53233)

gundalow committed:
Minor formatting (#53178)

Akasurde committed:
F5: correct check in _transform_api_names API (#53269)

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

felixfontein committed:
Make clear which option is used by which provider. (#53208)

Akasurde committed:
VMware: vmware_vm_inventory using vaulted config (#53048)

* VMware: vmware_vm_inventory using vaulted config

Adds documentation about using vaulted inventory cofiguration file
for vmware_vm_inventory plugin.

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

* Review comments

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

acozine commented on lib/ansible/modules/system/vdo.py:
```diff
@@ -127,6 +127,8 @@
               volumes will maintain their previously configured
               setting unless a different value is specified in the
               playbook.
+              The read cache feature is available in VDO 6.1 and
```
```suggestion
            - The read cache feature is available in VDO 6.1 and older.
```

acozine commented on lib/ansible/modules/system/vdo.py:
```diff
@@ -127,6 +127,8 @@
               setting unless a different value is specified in the
               playbook.
+              The read cache feature is available in VDO 6.1 and
+              older.
```
```suggestion
```

acozine commented on lib/ansible/modules/system/vdo.py:
```diff
@@ -142,6 +144,8 @@
               Existing volumes will maintain their previously
               configured setting unless a different value is specified
               in the playbook.
+              The read cache feature is available in VDO 6.1 and
```
```suggestion
           -  The read cache feature is available in VDO 6.1 and older.
```

acozine commented on lib/ansible/modules/system/vdo.py:
```diff
@@ -142,6 +144,8 @@
               configured setting unless a different value is specified
               in the playbook.
+              The read cache feature is available in VDO 6.1 and
+              older.
```
```suggestion
```

acozine reviewed (commented):
Prefer to call this out on a single, separate line for easier readability.

Akasurde committed:
VMware: Fail when unable to find managed object (#53158)

Fixes: #52119

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

carchi8py committed:
Add support for FlexCache to na_ontap_vserver_peer (#52301)

* Revert "changes to clusteR"

This reverts commit 33ee1b71e4bc8435fb315762a871f8c4cb6c5f80.

* updates

* doc fix

* Revert "Revert "changes to clusteR""

This reverts commit 8e56b999e6cf6a65de339e516f7134a6b6b39cba.

* fix wrong fucntion

carchi8py committed:
Bug fix for na_elementsw_admin_users (#52377)

* Revert "changes to clusteR"

This reverts commit 33ee1b71e4bc8435fb315762a871f8c4cb6c5f80.

* Revert "Revert "changes to clusteR""

This reverts commit f1104a37b42886aebb4d2b2ab27c91c96d97858a.

* Updates

* bug fix

* Bug fix

* Revert "Bug fix"

This reverts commit 5cbea70a6587b7bc5e8a4da912549fbc63ff6235.

samccann committed:
replaced outdate release info with checkmarks (#53016)

carchi8py committed:
Add NVME to gatherfacts (#53007)

* Revert "changes to clusteR"

This reverts commit 33ee1b71e4bc8435fb315762a871f8c4cb6c5f80.

* Revert "Revert "changes to clusteR""

This reverts commit f1104a37b42886aebb4d2b2ab27c91c96d97858a.

* Updtes

carchi8py committed:
Wrong Setup function was being used, which could result in failure if new global options were used.  (#52862)

* Revert "changes to clusteR"

This reverts commit 33ee1b71e4bc8435fb315762a871f8c4cb6c5f80.

* Revert "changes to clusteR"

This reverts commit 33ee1b71e4bc8435fb315762a871f8c4cb6c5f80.

* Revert "Revert "changes to clusteR""

This reverts commit f1104a37b42886aebb4d2b2ab27c91c96d97858a.

* Revert "Revert "changes to clusteR""

This reverts commit f1104a37b42886aebb4d2b2ab27c91c96d97858a.

* Fix setup bug

Akasurde committed:
Correct usages of module.warn (#53268)

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

mmazur committed:
k8s_auth: use group module defaults in example (#52712)

The yaml anchor thing was just a stopgap until all the module defaults
group stuff got merged. Which it did.

mkrizek committed:
yum/dnf: Add download_dir param (#53171)

s-hertel committed:
If the private_ip has been provided for the new network interface it shouldn't also be added to top level parameters for run_instances (#52579)

changelog

bgurney-rh committed:
Merge branch 'devel' of https://github.com/bgurney-rh/ansible into devel

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

Conflicts:
	lib/ansible/modules/system/vdo.py

mkrizek committed:
Clarify return value of the sequence lookup (#53275)

* Clarify return value of the sequence lookup

Fixes #17852

* Mention range filter

bgurney-rh committed:
vdo: add hyphens to readcache version annotation

Add leading hyphens to the version information for the "readcache"
and "readcachesize" options, to aid in readability.

Signed-off-by: Bryan Gurney <bgurney@redhat.com>

bgurney-rh committed:
Merge branch 'devel' of https://github.com/bgurney-rh/ansible into devel

bgurney-rh commented:
My repo is set up incorrectly; I'll abandon this PR and try again later.
